src/test/logic/token/utils/formatAmount.test.ts

```diff
@@ -0,0 +1,167 @@
+import { formatAmountInUsFormat, formatAmount } from '../../../../logic/tokens/utils/formatAmount'
+
+describe('formatAmount', () => {
+  it('Given 0 returns 0',  () => {
+    // given
+    const input = '0'
+    const resultExpected = '0'
+
+    // when
+    const result = formatAmount(input)
+
+    // then
+    expect(result).toBe(resultExpected)
+  })
+  it('Given 1 returns 1',  () => {
+    // given
+    const input = '1'
+    const resultExpected = '1'
+
+    // when
+    const result = formatAmount(input)
+
+    // then
+    expect(result).toBe(resultExpected)
+  })
+  it('Given 19797.899 returns 19,797.899',  () => {
```

Comment:
lets rewrite test cases differently:
`Given a number between 10000 and 100000 returns a number of format XX,XXX.XXX`
 (apply the same paradigm to all cases)